cobalt/h5vcc/h5vcc_settings.cc

```diff
@@ -104,5 +104,22 @@ bool H5vccSettings::Set(const std::string& name, SetValueType value) const {
   return false;
 }
 
+void H5vccSettings::SetPersistentSettingAsInt(const std::string& key,
+                                              int value) const {
+  if (persistent_settings_) {
+    persistent_settings_->SetPersistentSetting(
+        key, std::make_unique<base::Value>(value));
+  }
+}
+
+int H5vccSettings::GetPersistentSettingAsInt(const std::string& key,
+                                             int default_setting) const {
+  if (persistent_settings_) {
+    return persistent_settings_->GetPersistentSettingAsInt(key,
+                                                           default_setting);
+  }
+  return -1;
```

Comment:
+1

Reply:
I thought about returning default_setting so that we would fully be a pass through to the underlying setters/getters. Doing so will not allow us to see cases where the call fails due to the `persistent_settings_` object not yet being initialized.